### ISSUE TITLE
Bot doesn't seem to reconnect sometimes after being destroyed

### ISSUE THREAD

**Issue by Escobar805** [Alph4rd/Toshinou-Ultra#11]
I run the bot on a VPS so don't watch it all the time. Sometimes I do open up the the VPS and see that the game is disconnected and after checking the logbook it seems to be destroyed by another player each time.

**Comment by Alph4rd** [comment]
Yeah,  i've been experiencing similar issues, the bot doesn't seem to reconnect sometimes.
Not sure what the problem is.
